system-parachains/collectives/collectives-polkadot/src/secretary/mod.rs

```diff
@@ -0,0 +1,294 @@
+/// of members of the Secretary Collective.
+type OpenGovOrSecretaryOrFellow = EitherOfDiverse<
+	EitherOfDiverse<EnsureRoot<AccountId>, Fellows>,
+	EitherOfDiverse<Secretary, EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>>,
+>;
+
+/// Origins of:
+/// - Root;
+/// - FellowshipAdmin (i.e. token holder referendum);
+/// - Plurality vote from Fellows can exchange origins and
+/// - configure the parameters that govern the Collective.
+type OpenGovOrFellow = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	EitherOfDiverse<Fellows, EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>>,
+>;
+
+impl pallet_secretary_origins::Config for Runtime {}
+
+pub type SecretaryReferendaInstance = pallet_referenda::Instance3;
+
+impl pallet_referenda::Config<SecretaryReferendaInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_referenda_secretary_referenda::WeightInfo<Runtime>;
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type Scheduler = Scheduler;
+	type Currency = Balances;
+	// Secretary collective can submit proposals.
+	type SubmitOrigin = pallet_ranked_collective::EnsureMember<
+		Runtime,
+		SecretaryCollectiveInstance,
+		{ ranks::SECRETARY },
+	>;
+	type CancelOrigin = OpenGovOrSecretaryOrFellow;
+	type KillOrigin = OpenGovOrSecretaryOrFellow;
+	type Slash = ToParentTreasury<PolkadotTreasuryAccount, LocationToAccountId, Runtime>;
+	type Votes = pallet_ranked_collective::Votes;
+	type Tally = pallet_ranked_collective::TallyOf<Runtime, SecretaryCollectiveInstance>;
+	type SubmissionDeposit = ConstU128<0>;
+	type MaxQueued = ConstU32<100>;
+	type UndecidingTimeout = ConstU32<{ 7 * DAYS }>;
+	type AlarmInterval = ConstU32<1>;
+	type Tracks = tracks::TracksInfo;
+	type Preimages = Preimage;
+}
+
+pub type SecretaryCollectiveInstance = pallet_ranked_collective::Instance3;
+
+impl pallet_ranked_collective::Config<SecretaryCollectiveInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_ranked_collective_secretary_collective::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type AddOrigin = OpenGovOrFellow;
+	type RemoveOrigin = ApproveOrigin;
+	type PromoteOrigin = ApproveOrigin;
+	type DemoteOrigin = ApproveOrigin;
+	type ExchangeOrigin = OpenGovOrFellow;
+	type Polls = SecretaryReferenda;
+	type MinRankOfClass = Identity;
+	type MemberSwappedHandler = crate::SecretarySalary;
+	type VoteWeight = pallet_ranked_collective::Geometric;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkSetup = crate::SecretarySalary;
+}
+
+pub type SecretarySalaryInstance = pallet_salary::Instance3;
+
+parameter_types! {
+	// The interior location on AssetHub for the paying account. This is the Secretary Salary
+	// pallet instance. This sovereign account will need funding.
+	pub SecretarySalaryInteriorLocation: InteriorLocation = PalletInstance(<crate::SecretarySalary as PalletInfoAccess>::index() as u8).into();
+}
+
+const USDT_UNITS: u128 = 1_000_000;
```

Comment:
Are we copying this into every collective?

Reply:
Yeah, `USDT_UNITS` is the same in [Ambassador Collective](https://github.com/polkadot-fellows/runtimes/blob/7723274a2c5cbb10213379271094d5180716ca7d/system-parachains/collectives/collectives-polkadot/src/ambassador/mod.rs#L226) & the [Technical Fellowship](https://github.com/polkadot-fellows/runtimes/blob/7723274a2c5cbb10213379271094d5180716ca7d/system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs#L225).

Reply:
Can we just move it to some common location and use it from there?